Makefile

```diff
@@ -13,14 +13,14 @@ load-design-system-templates:
 	pipenv run ./scripts/load_templates.sh $(DESIGN_SYSTEM_VERSION)
 
 lint:
-	pipenv check ./response_operations_ui ./tests
+	pipenv check ./response_operations_ui ./tests -i 51457 -i 51499
```

Comment:
You can remove this as the issues are fixed now

Reply:
This has been removed.